decompiler/ff7_field/ff7_field_engine.cpp

```diff
@@ -1600,6 +1880,69 @@ void FF7::FF7CameraInstruction::processInst(Function& func, ValueStack&, Engine*
     }
 }
 
+void FF7::FF7CameraInstruction::processNFADE(CodeGenerator* codeGen)
```

Comment:
That's gonna be some serious class explosion. If you're cool with that then sure, but I'll shelve it for a cleanup day.